pyomo/util/subsystems.py

```diff
@@ -38,26 +55,45 @@ def exitNode(self, node, data):
     def finalizeResult(self, result):
         return self._functions
 
-    def enterNode(self, node):
-        pass
+    # def enterNode(self, node):
+    #    pass
 
-    def acceptChildResult(self, node, data, child_result, child_idx):
-        pass
-
-    def acceptChildResult(self, node, data, child_result, child_idx):
-        if child_result.__class__ in native_types:
-            return False, None
-        return child_result.is_expression_type(), None
+    # def acceptChildResult(self, node, data, child_result, child_idx):
+    #    if child_result.__class__ in native_types:
+    #        return False, None
+    #    return child_result.is_expression_type(), None
```

Comment:
Should these be deleted?

Reply:
I included these for some supposed performance benefit in the initial implementation of `_ExternalFunctionVisitor`, but I actually notice a small performance improvement (about 0.1 s in the external function visitor) on my motivating test problem when I remove them. I will remove these.